web/admin/application/configs/klear/klear.yaml

```diff
@@ -53,191 +53,212 @@ production:
+            CompanyBalancesList:
+              title: ngettext('Prepaid Balance', 'Prepaid Balances', 0)
+              class: ui-silk-money-yen
+              description: _("List of %s", ngettext('Prepaid Balance', 'Prepaid Balances', 0))
+        Invoicing:
+          title: _('Invoicing')
           class: ui-silk-report
-          description: _("List of %s", ngettext('Invoice', 'Invoices', 0))
           showOnlyIf: ${auth.brandFeatures.invoices.enabled}
-        BillableCallsBrandList:
-          title: ngettext('Billable call', 'Billable calls', 0)
-          class: ui-silk-application-view-list
-          description: _("List of %s", ngettext('Billable call', 'Billable calls', 0))
-          showOnlyIf: ${auth.brandFeatures.billing.enabled}
-        CallCsvSchedulersBrandList:
-          title: ngettext('Call CSV scheduler', 'Call CSV schedulers', 1)
-          class: ui-silk-application-view-list-schedule
-          description: _("List of %s", ngettext('Call CSV scheduler', 'Call CSV schedulers', 0))
+          submenus:
+            InvoicesList:
+              title: ngettext('Invoice', 'Invoices', 0)
+              class: ui-silk-report
+              description: _("List of %s", ngettext('Invoice', 'Invoices', 0))
+            InvoiceSchedulersList:
+              title: ngettext('Invoice scheduler', 'Invoice schedulers', 0)
+              class: ui-silk-report-clock
+              description: _("List of %s", ngettext('Invoice scheduler', 'Invoice schedulers', 0))
+            InvoiceNumberSequencesList:
+              title: ngettext('Invoice number sequence', 'Invoice number sequences', 0)
+              class: ui-silk-report-go
+              description: _("List of %s", ngettext('Invoice number sequence', 'Invoice number sequences', 0))
+            FixedCostsList:
+              title: ngettext('Fixed cost', 'Fixed costs', 0)
+              class:  ui-silk-money-add
+              description: _("List of %s", ngettext('Fixed cost', 'Fixed costs', 0))
+            InvoiceTemplatesList:
+              title: ngettext('Invoice template', 'Invoice templates', 0)
+              class: ui-silk-report-key
+              description: _("List of %s", ngettext('Invoice template', 'Invoice templates', 0))
+        CallListsBrand:
+          title: _("Calls")
+          class: ui-silk-application-view-list-telephone
+          submenus:
+            BillableCallsBrandList:
+              title: ngettext('Billable call', 'Billable calls', 0)
+              class: ui-silk-application-view-list-money
+              description: _("List of %s", ngettext('Billable call', 'Billable calls', 0))
+            CallCsvSchedulersBrandList:
+              title: ngettext('Call CSV scheduler', 'Call CSV schedulers', 1)
+              class: ui-silk-application-view-list-schedule
+              description: _("List of %s", ngettext('Call CSV scheduler', 'Call CSV schedulers', 0))
+        Misc:
```

Comment:
Following items are Global brand items:

- Portal URLs
- Notification Templates
- Generic Music On Hold
- Services
- Generic Match Lists

Following items are Brand *views* of client items:
- DDIs
- Retail Accounts
- Residential Devices

We could split this menu into too different ones, or leave them as is